src/cli_config.rs

```diff
@@ -117,16 +201,14 @@ pub fn cli_parse() -> CliConfig {
             .short("u")
             .long("udp-address")
             .help("UDP address to run DHT node")
-            .takes_value(true)
-            .required_unless("tcp-address"))
+            .takes_value(true))
```

Comment:
Can we retain these requirements? It should not be required only in case when config is specified.

Reply:
done